pkg/ddc/goosefs/transform_api_gateway_test.go

```diff
@@ -1,3 +1,17 @@
+Copyright <year> The <project-name> Author.
```

Comment:
1. This should be `Copyright 2022 The Fluid Authors.`
2. Please use block comment to add license header